src/app/(header-footer)/error.tsx

```
@@ -0,0 +1,16 @@
+'use client';
+
+interface ErrorProps {
+  error: Error & { digest: string };
+  reset: () => void;
+}
+
+// TODO: 나중에 디자인 하기
+export default function Error({ error, reset }: ErrorProps) {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)